### PR TITLE
Add project banner

### PR DESCRIPTION
**Pull request by DexLulz**
I think this could be a cool banner to add to the tool just to flare it up a little bit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by DexLulz** [comment]
It wasn't a logo I used a photograph of a spider, its on my laptop I can get the original later today if you would like, your tool looks great btw!

**Comment by PROxZIMA** [comment]
> I used a photograph of a spider
Wohh, I thought it was a logo because the ascii looks so clean.

> your tool looks great btw
Haha, it still lacks tons of features which are mentioned in #23. We are actively working on them.

About this PR, I'll merge it after adding a proper "display in terminal" like banner.

File: darkspider.py [hunk 1]
```diff
@@ -1,6 +1,33 @@
 #!/usr/bin/env python3
 
+#I think this banner could be cool for the tool If you like it delete this comment
```
Comment (PROxZIMA) [inline darkspider.py:3]: You can delete this comment.

**Review by PROxZIMA — CHANGES_REQUESTED** [review]

File: darkspider.py (hunk 1 above)
Comment (DexLulz) [reply darkspider.py]: Done

**Review by PROxZIMA — APPROVED** [review]

**Comment by PROxZIMA** [comment]
Finally got some free time to look into this. Few changes:
- Super cool gradient banner.
image
- Colored logger
- Helper module is more modular now.

Just need to test it on windows and it'll be ready to merge!!

**Pull request merged**